app/controllers/participants_controller.rb

```diff
@@ -40,6 +41,8 @@ def index
     @buses = Bus.order :number
   end
 
+  def scoreboard; end
```

Comment:
I thought Rails 5 didn't need empty controller actions.

Reply:
It doesn't. But Rubocop doesn't like the presence of actions in `before_action`s that may or may not exist, and I _think_ I agree with it. Because the controller does have a non-default effect on the request (in the form of its `before_action`s)

Seems a little like needless cruft, but is also a bit of a breadcrumb for future developers, perhaps?

Reply:
Sure, seems reasonable - though I would suggest that a comment to that effect would be a bigger breadcrumb.